cime_config/machines/config_machines.xml

```diff
@@ -659,6 +659,193 @@
     </environment_variables>
   </machine>
 
+
+
+
+  <machine MACH="crusher-cpu">
```

Comment:
This really ought to go into upstream, but it won't conflict because upstream defines only "crusher", so I'm OK with this for now. Later, if E3SM needs to define crusher-cpu and crusher-gpu, we might need to reconcile the upstream and SCREAM versions.

Reply:
yes, i believe actually that e3sm right now does not want the workaround like crusher-cpu and crusher-gpu, instead, the'd use only compiler name to distinguish. for me it was easier to do it this way to avoid bugs. this will be sorted later.